packages/core/src/html/CustomListIconProcessor.ts

```diff
@@ -138,12 +143,42 @@ function updateLi(node: MbNode, iconAttributes: IconAttributes) {
 // See https://github.com/MarkBind/markbind/pull/2316#discussion_r1255364486 for more details.
 function handleLiNode(node: MbNode, iconAttrValue: IconAttributeDetail) {
   if (iconAttrValue.isFirst) {
-    iconAttrValue.iconAttrs = getIconAttributes(node);
+    const nodeIconAttrs = getIconAttributes(node);
+    // Check if first item is customized
+    if (nodeIconAttrs?.icon !== undefined) {
+      iconAttrValue.addIcons = true;
+    }
+    // Save if the icon is not one-off
+    if (nodeIconAttrs?.oneOff === undefined) {
+      iconAttrValue.iconAttrs = nodeIconAttrs;
+    }
     iconAttrValue.isFirst = false;
   } else if (iconAttrValue.iconAttrs) {
-    iconAttrValue.iconAttrs = getIconAttributes(node, iconAttrValue.iconAttrs);
+    const nodeIconAttrs = getIconAttributes(node, iconAttrValue.iconAttrs);
+    // Save if the icon is not one-off
+    if (nodeIconAttrs?.oneOff === undefined) {
+      iconAttrValue.iconAttrs = nodeIconAttrs;
+    }
+  }
+  if (iconAttrValue.addIcons === false) {
```

Comment:
```suggestion
  if (!iconAttrValue.addIcons) {
```
Hi Kevin, please DO NOT do this again. This is very bad code quality.